frontend/locales/nl.json

```
@@ -29,7 +29,7 @@
   "Please Note: Accepted applications go to the archive. See above menu. The archive is available for everyone at the same facility with with a Solid-ID. " : "NB: Goedgekeurde aanvragen komen in het archief van deze portal te staan, zie het menu hierboven. Dit archief is toegankelijk voor iedereen met een Solis-ID die bij dezelfde faculteit werkt of studeert.",
   "UU-webpage of the FEtC-H": "UU-webpagina van de FETC-GW",
   "Regulations of the FEtC-H": "Reglement van de FETC-GW",
-  "Model documents": "Voorbeelddocumenten",
+  "Model documents": "Voorbeeld documenten",
```

```suggestion
  "Model documents": "Voorbeelddocumenten",
```

The original spelling without a space is correct.

---

Are Model Documents still two words in that case?